plugins/source/azure/resources/services/servicebus/topic_authorization_rules.go

```diff
@@ -0,0 +1,73 @@
+package servicebus
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/servicebus/armservicebus"
+	"github.com/cloudquery/cloudquery/plugins/source/azure/client"
+	"github.com/cloudquery/plugin-sdk/faker"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+	"github.com/gorilla/mux"
+)
+
+func topicAuthorizationRules() *schema.Table {
+	return &schema.Table{
+		Name:        "azure_servicebus_namespace_topic_authorization_rules",
```

Comment:
There are also auth rules at the namespace level (which are different) hence the prefixing